src/nvim/os/unix_defs.h

```diff
@@ -28,6 +28,9 @@
 #ifndef SYNTAX_FNAME
 # define SYNTAX_FNAME   "$VIMRUNTIME/syntax/%s.vim"
 #endif
+#ifndef SYNTAX_FILE
+# define SYNTAX_FILE   "$VIMRUNTIME/syntax/syntax.vim"
+#endif
```

Comment:
This was rebased onto master last week, I didn't see any changes related to this.